jdisc_core/src/main/java/com/yahoo/jdisc/core/ApplicationLoader.java

```diff
@@ -186,7 +186,7 @@ public void stop() throws Exception {
         synchronized (appLock) {
             application = null;
         }
-        activateContainer(null);
+        try (DeactivatedContainer deactivated = activateContainer(null)) { }
```

Comment:
Can be simplified to `activacteContainer(null).close()`

Reply:
`activateContainer(...)` returns `null` when there is no previous container.

Reply:
Clever!